docs/troubleshooting/authentication/custom_authentication_in_flutterFlow.md

```diff
@@ -0,0 +1,79 @@
+---
+keywords: ['authentication', 'api', 'configuration']
+slug: /custom-authentication-in-flutterflow
+title: Custom Authentication in FlutterFlow
+---
+# Custom Authentication in FlutterFlow
+
+**Pre-Requisites to Enabling Custom Authentication:**
+
+Ensure you have a **custom server** with login and sign-up endpoints that return a JWT token upon success
+
+**Custom authentication** must be enabled in FlutterFlow, with entry and logged-in pages correctly set
+
+Here's an example: 
+
+![](../../assets/20250430121149388590.png)
+
+**Checklist for Troubleshooting**
+
+**Verify Server and API Endpoints**
+
+Confirm that your server is correctly returning JWT tokens for login and sign-up requests. The server's response should include the authentication token, refresh token, expiration time, and user ID (UID).
```

Comment:
bullets are missing from this doc completely, please revisit this file and add bullets or numbering wherever necessary.